caikit/runtime/service_factory.py

```diff
@@ -59,42 +62,6 @@
 log = alog.use_channel("SVC-FACTORY")
 error = error_handler.get(log)
 
-TRAINING_MANAGEMENT_SERVICE_NAME = "TrainingManagement"
```

Comment:
It turns out, all of this had been half-way moved to `names.py`, but never removed from here. I consolidated it all over there.